### PR TITLE
Folder permissions

### PR DESCRIPTION
**Pull request by CrystalPea**
Part of this story: https://www.pivotaltracker.com/story/show/164632434

### CODE REVIEW

File: tests/functional/preview_and_dev/test_seeded_user.py [hunk 1]
```diff
@@ -334,6 +338,84 @@ def test_creating_moving_and_deleting_template_folders(driver, login_seeded_user
     assert template_name not in [x.text for x in driver.find_elements_by_class_name('message-name')]
 
 
+def test_template_folder_permissions(driver, login_seeded_user):
+    folder_names = [
+        'test-parent-folder {}'.format(uuid.uuid4()),
+        'test-child-folder {}'.format(uuid.uuid4()),
+        'test-grandchild-folder {}'.format(uuid.uuid4()),
```
Comment (leohemsted) [inline tests/functional/preview_and_dev/test_seeded_user.py]: do we want these to have the same uuid? (as a kind of "test run" sort of thing)

not sure it makes a huge difference, just wonder if it'll make it easier to debug/search if a test fails

File: tests/pages/pages.py [hunk 2]
```diff
@@ -586,6 +628,13 @@ def click_invite_user(self):
         element = self.wait_for_element(TeamMembersPage.invite_team_member_button)
         element.click()
 
+    def click_edit_team_member(self, email=None):
```
Comment (leohemsted) [inline tests/pages/pages.py]: we never call this without passing `email` in, so i think we can get rid of this custom url

Comment (CrystalPea) [reply tests/pages/pages.py]: ahh, you are right, probably it was a leftover 😅. Fixed now :)

File: tests/functional/preview_and_dev/test_seeded_user.py (hunk 1 above)
Comment (CrystalPea) [reply tests/functional/preview_and_dev/test_seeded_user.py]: Good suggestion, I changed them to have common id now :)

**Pull request merged**